test/functional/test.js

```diff
@@ -0,0 +1,63 @@
+import { Selector, t } from 'testcafe';
```

Comment:
We usually split all imports in our code files to improve readability
first 3d party modules 
then a blank line
then local files